### ISSUE TITLE
Accessibility: Distinct Hit/Hover States for Pick List and Value List

### ISSUE THREAD

**Issue by bstifle** [Esri/calcite-components#1547]
Currently the hover state triggers the whole item and action background transition. This is an outlier for the Calcite Component system, as actions should have distinct hover/hit states from other elements. This is an accessibility concern as it is confusing when trying to make a decision between the item selection and the overflow/action area.

Current vs proposed design:

![image](https://user-images.githubusercontent.com/14083530/107817260-8235e400-6d3b-11eb-8c30-f7c900206bdb.png)

**Comment by julio8a** [comment]
Bryan, is this one still valid? if so, can we add these requirements and close this issue?

**Comment by bstifle** [comment]
@julio8a yes still valid. Mike Stinson is working on it with his PR #2502

**Comment by caripizza** [comment]
@rmstinson are you still working on this one or was this covered in your pr #2502?
